browser/resources/settings/brave_clear_browsing_data_dialog/brave_clear_browsing_data_dialog_behavior.html

```diff
@@ -0,0 +1,4 @@
+<link rel="href" src="chrome://resources/html/cr.html">
```

Comment:
I don't think this is valid (`rel="href"`) and I don't think cr.html is needed here, but if so shouldn't it be `<link rel="import" href="...`?

Reply:
For some reason _proxy.html files use that syntax, which is where I copied it from.

Reply:
Removed. It was, indeed, not needed.